components/match2/wikis/leagueoflegends/legacy/match_group_legacy.lua

```diff
@@ -0,0 +1,250 @@
+---
```

Comment:
[diff](https://github.com/Liquipedia/Lua-Modules/blob/8fbc2c311d3cae592a8e7d738fcfc26a137a8443/components/match2/wikis/leagueoflegends/legacy/match_group_legacy.lua#L212-L218)
Tbh I don't know what is the best way to handle this, because each map inside BracketMatchSummary is a Json.
![image](https://github.com/Liquipedia/Lua-Modules/assets/43279191/eab24f70-31c1-42d2-b0d6-0d73cf725d3e)
[sandbox](https://liquipedia.net/leagueoflegends/User:LuckeLucky/sandbox/2)

Reply:
using spaces over tabs (should be tabs)
![IMG_4833](https://github.com/Liquipedia/Lua-Modules/assets/75081997/403d4ba6-0884-4cc4-b17a-bbbc644889ef)

also how about doing that call in a sep. module that gets called instead of the json one?
that way could probably drop the overwrite of this module
(might need adjusts to the matchlist one though)

Reply:
`match[realKey] = MatchSubobjects.luaGetMap(nestedArgs)` is called in the commons version, and idk how to handle `matchX`(mapX) without parsing it